ui/app/templates/workspace/projects/project/app/deployments.hbs

```diff
@@ -15,20 +19,20 @@
         from the CLI</p>
     </EmptyState>
   {{/each}}
-  {{#if (and this.hasMoreDeployments (not this.destroyed))}}
-    <div class="destroyed-deployments-filter">
-      <Pds::HelpText>
-        Some deployments have been destroyed and are hidden.
-        <Pds::Button
-          @compact={{true}}
-          @variant="ghost"
-          {{on "click" this.showDestroyed}}>
-          Display destroyed deployments
-        </Pds::Button>
-      </Pds::HelpText>
-    </div>
-  {{/if}}
-
 </ul>
+{{#if (and this.hasMoreDeployments (not this.isShowingDestroyed))}}
+  <div class="destroyed-deployments-filter">
+    <Pds::HelpText>
+      {{t "page.deployments.destroyed_and_hidden_state"}}
+      <Pds::Button
+        data-test-display-destroyed-button
+        @compact={{true}}
+        @variant="ghost"
+        {{on "click" this.showDestroyed}}>
+        Display destroyed deployments
```

Comment:
I'm not sure if we're attempting to move all strings to `{{t ...` but if so there's another case here